spec/modules-spec.js

```diff
@@ -24,6 +25,21 @@ describe('modules support', () => {
           done()
         })
       })
+      if (process.platform === 'win32') {
+        it('can be required if electron.exe is renamed', () => {
+          const testExecPath = path.join(path.dirname(remote.process.execPath), "test.exe")
+          fs.copyFileSync(process.execPath, testExecPath)
+          try {
+            const runasFixture = path.join(fixtures, 'module', 'runas-renamed.js')
+            assert.ok(fs.existsSync(runasFixture))
+            const child = require('child_process').spawnSync(testExecPath, [runasFixture])
+            assert.strictEqual(child.stdout.toString().trim(), "good")
```

Comment:
This will break when electron decides to log extra things for whatever reason, can we test exit codes instead